CHANGELOG.md

```diff
@@ -41,7 +41,8 @@ export class IgxCustomFilteringOperand extends IgxFilteringOperand {
 
 - **Breaking changes**:
     - Removed submodule imports. All imports are now resolved from the top level `igniteui-angular` package.
-    - `igxToggle` changes
+    - `igxToggle` changes:
+        - `igxToggle` is renamed to `igxOverlay`.
```

Comment:
Wait, are you sure we need to rename the toggle? I don't quite remember when this decision was made!